tests/geomultiscale-mapping/RadialGeoMultiscale.cpp

```diff
@@ -18,61 +18,66 @@ BOOST_AUTO_TEST_CASE(RadialGeoMultiscale)
 
   Participant cplInterface(context.name, context.config(), 0, 1);
   if (context.isNamed("SolverOne")) {
-    auto     meshName = "MeshOne";
-    Vector3d vec1     = Vector3d::Constant(0.0);
-    auto     vid      = cplInterface.setMeshVertex(meshName, vec1);
-    auto     dataAID  = "DataOne";
-    auto     dataBID  = "DataTwo";
+    Vector3d            coordOneA{0.0, 0.0, 0.0};
+    Vector3d            coordOneB{0.0, 0.0, 1.0};
+    Vector3d            coordOneC{0.0, 0.0, 2.0};
+    std::vector<double> values;
+    const unsigned int  nCoords = 3;
+    for (unsigned int i = 0; i < nCoords; ++i) {
+      values.emplace_back(std::pow(i + 1, 2));
+    }
+    auto             meshOneID = "MeshOne";
+    std::vector<int> ids;
+    ids.emplace_back(cplInterface.setMeshVertex(meshOneID, coordOneA));
+    ids.emplace_back(cplInterface.setMeshVertex(meshOneID, coordOneB));
+    ids.emplace_back(cplInterface.setMeshVertex(meshOneID, coordOneC));
+
+    auto dataAID = "DataOne";
 
-    double valueDataB = 1.0;
     cplInterface.initialize();
     double maxDt = cplInterface.getMaxTimeStepSize();
-    cplInterface.readData(meshName, dataBID, {&vid, 1}, maxDt, {&valueDataB, 1});
-    BOOST_TEST(valueDataB == 1.0);
-
-    while (cplInterface.isCouplingOngoing()) {
+    BOOST_TEST(cplInterface.isCouplingOngoing());
```

Comment:
This does not look right, does it? I can imagine that the test gets stuck here.

Reply:
The error was in the xml file, where I specified too many time windows.